lib/subspace/commands/base.rb

```diff
@@ -53,6 +53,23 @@ def confirm_overwrite(file_path)
         return answer.downcase.start_with? "y"
       end
 
+      def pass_through_params
+        ansible_options = []
+        self.class::PASS_THROUGH_PARAMS.each do |param_name|
```

Comment:
fancy, i wasn't sure how this could be done :+1: i didn't realize this was valid syntax lol